### ISSUE TITLE
VRMS: All Team Meeting Agenda and Meeting Minutes

### ISSUE THREAD

**Issue by drubgrubby** [hackforla/VRMS#1051]
### Overview
This issue tracks the agenda for the VRMS Team's Monday meeting

#### Pre-work to prep for meeting
- [ ] Update issues

#### Recurring items: 
- [ ] Review project board
- [ ] Assign issues

#### Previous Meetings (link to each comment with the date)
[February 14, 2022](https://github.com/hackforla/VRMS/issues/1051#issuecomment-1032187920)
[February 21, 2022](https://github.com/hackforla/VRMS/issues/1051#issuecomment-1040404276)
[March 7, 2022](https://github.com/hackforla/VRMS/issues/1051#issuecomment-1061388952)

**Comment by drubgrubby** [comment]
#### February 14, 2022  Meeting Agenda

#### Pre-work to prep for meeting
- [ ] Update issues

### Who was there?
David, Walter, Shaun, Glen, Evan and Bonnie for a second

#### Recurring items: 
- [x] Review project board
- [x] Assign issues

#### Things we plan to discuss
- [x] https://github.com/hackforla/VRMS/issues/1048
- [x] Discuss node-sass PRs
- [x] adding NODE_ENV=test to your backend .env file
- [x] How to comment when creating a new PR

#### What we actually discussed
- [ ] Bonnie explained what she wants on #1048 
- [ ] Comment out slack call so we don't get the error anymore (create ticket)
-[ ] We need to create a ticket to make sure that the production db has the user.email field as a unique value

#### Additional notes

#### Upcoming Milestones

#### Previous Meetings (link to each comment with the date)

**Comment by drubgrubby** [comment]
#### February 21, 2022  Meeting Agenda

#### Pre-work to prep for meeting
- [ ] Update issues

### Who was there?
Shaun, Evan, David

#### Recurring items: 
- [ ] Review project board
- [ ] Assign issues

#### Things we plan to discuss
- [ ] https://github.com/hackforla/VRMS/issues/1065
- [ ] Discuss the use of Prettier / eslint for code consistancy

#### What we actually discussed
- We need to discuss with Bonnie how to think about merging users so we can remove duplicate email addresses from the database. 
- We're going to suffer through the ugly prettier PRs for a minute for the greater glory of having our code look prettier. 
- We should add an eslint pipeline step that fails on eslint errors
- Also we should update the readme to mention installing eslint and prettier ide plugins (Create a ticket for that). 
- Create a ticket for having db calls refer to new endpoints

#### Additional notes

#### Upcoming Milestones

**Comment by drubgrubby** [comment]
#### Monday, February 28, 2022  Meeting Agenda

#### Pre-work to prep for meeting
- [x] Update issues

### Who was there?

#### Recurring items: 
- [ ] Review project board
- [ ] Assign issues

#### Things we plan to discuss
-[ ] What is there still to do to get v3 done? 

#### What we actually discussed

**Comment by drubgrubby** [comment]
#### Monday, March 7, 2022  Meeting Agenda

#### Pre-work to prep for meeting
- [ ] Update issues

### Who was there?
Jeanette, Josh, Shaun, Evan, Haira
#### Recurring items: 
- [ ] Review project board
- [ ] Assign issues

#### Things we plan to discuss


#### What we actually discussed
- We discussed with Jeanette the data that she is looking for and how to get it. 
- We discussed the issue of making the user.email field in the production database unique.  There are two prevailing theories: 1) Make a copy of the users database so we have all of the information somewhere, and then make the change to the new main users db, 2) write a script that will append something to the duplicate users email, ex. 
- We talked about the auth PR from v.4 that Shaun is going to take a look at. 

#### Additional notes

#### Upcoming Milestones

**Comment by jbubar** [comment]
### VRMS Tech meeting
- went over Evan's PR and decided that he could do it again with out bootstrap.
- scheduled a meeting with josh and Walter to work on walter's issue
- created a new issue for the annoying magic link https://github.com/hackforla/VRMS/issues/1080
    - evan's idea

**Comment by jbubar** [comment]
## VRMS Team Meeting 

Questions and requests for Bonnie:
- Can we give @FoxShaunR access to the AWS 
    - Ops is not responding (has been over a month)
- What do we need to deliver to get v0.3 out the door?
    - current list of things to do
        - Clean up the duplicate email addresses in the db https://github.com/hackforla/VRMS/issues/1065
            - Back up database, clean up prod database and keep newest one, update newer registration missing data with data from older registration (including login history!!), delete older registration
        - Restyle the admin portal to look like the figma https://github.com/hackforla/VRMS/issues/1049 (v.3.1)
        - Create github issue when someone wants to edit a restricted field in the projects info https://github.com/hackforla/VRMS/issues/796
        - ... **(unfinished)**
- Can someone be promoted to Admin to make github secrets so that we can make the deployment github actions work
- Can we get more thorough requirements for this issue?(https://github.com/100Automations/futureautomations/issues/8)
    - @tmshkr is interested in taking this on
    -

**Comment by jbubar** [comment]
Cognito party on the 21st

**Comment by ExperimentsInHonesty** [comment]
Please add this to this week's agenda and get it done during the meeting
- [ ] Taking a picture of the team [CTA Instructions](https://docs.google.com/presentation/d/1UDHwTQ1C3y01CiNS0mvPjNBHJFlm2Txs/preview)

**Comment by jbubar** [comment]
Meeting today Jun 13th.

@evanyang1  is going to take on an issue about removing the open close button that does not work - https://github.com/hackforla/VRMS/projects/2#card-83026620
@sarL3y  is taking on the issue where we are separating v.3 and v.4 into seperate git branches - https://github.com/hackforla/VRMS/issues/1116

Looks like we are going to have a meeting on the 29th to deploy

we just have a few things left before we deploy v0.3 and move all our energy to v0.4

**Comment by jbubar** [comment]
#### Monday, July 10, 2022  Meeting Agenda

#### Pre-work to prep for meeting
- [ ] Update issues

### Who was there?

#### Things we plan to discuss
- [ ] Review [v0.3 project board](https://github.com/hackforla/VRMS/projects/2)
    - [ ] In progress column
    - [ ] prioritized backlog
        - [ ] Make sure that everyone has an issue
        - [ ] Have a discussion on which issues need to be done before moving to v0.4, and which ones could potentially never be completed


#### What we actually discussed

#### Additional notes

#### Upcoming Milestones

**Comment by jbubar** [comment]
Closing this issue, because we have migrated our meeting notes to the wiki page: https://github.com/hackforla/VRMS/wiki/Team-Meetings
